docs/Help.md

```diff
@@ -98,3 +98,26 @@ This is usually a case because Chatterino is unable to save settings on your dis
 2. Back up your settings by copying them from your [Chatterino folder](#where-is-chatterino-folder-located) to a safe location on your disk and delete them. Restart your computer and try launching Chatterino again.
 3. Back up your settings, uninstall Chatterino and install it again, but make sure `Fresh Install` option is checked.
 4. Try running Chatterino in portable mode.
+
+### What is nightly and how to use / install it
+
+##### Overview
+Nightly is a "developer release" of Chatterino. It is released every time there's a new change added to source code. It includes some features that may not be available in the latest stable release.
+
+
+##### Installation
+1. Go to [nightly release page](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) on GitHub.
+1. Download nightly:
+    - For **Windows** download `test-cmake-chatterino-windows-x86-64.zip`
+    - For **Linux** download `test-cmake-Chatterino-x86_64.AppImage`
+    - For **Mac** download `test-cmake-chatterino-osx.dmg`
```

Comment:
```suggestion
    - For **Mac**, download `test-cmake-chatterino-osx.dmg`
```

Reply:
Since those are very simple sentences I don't feel like comma is necessary in here 😕